com.unity.visualeffectgraph/Shaders/ParticleLinesSW/PassForward.template

```diff
@@ -13,7 +13,7 @@ Pass
 	{
 		float4 pos : SV_POSITION;
 		#if IS_TRANSPARENT_PARTICLE
-#ifdef SHADER_API_PSSL
+#if defined (SHADER_API_PSSL) || defined(SHADER_API_GAMECORE)
```

Comment:
Why here we have gamecore and not xbox? I did not look at the reference case, but I wonder if it was PSSL specific only?

Reply:
Doesn't seem to be an issue on Xbox, but GPU hangs on GameCore

Reply:
+1 This is probably a common architecture issue, we are still investigating.